private-templates-service/server/package.json

```diff
@@ -6,23 +6,22 @@
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
-    "start": "node dist/server/src/server.js",
+    "start": "node dist/server.js",
     "build": "",
     "deploy": "tsc",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "server": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec ts-node src/server.ts | bunyan",
-    "client": "cd ../client && npm start",
-    "link-client": "cd ../adaptivecards-templating-client/typescript && npm install && npm run build && npm link && cd ../../client && npm link adaptive-templating-service-typescript-node && npm install && cd src && npm link adaptive-templating-service-typescript-node",
-    "init-app": "cd ../adaptivecards-templating-service && npm install && npm run build && npm link && cd ../server && npm link adaptivecards-templating-service && npm install && npm run link-client"
+    "client": "cd ../client && npm start"
```

Comment:
do we only have to `npm install` and `npm run dev` to run the whole thing from scratch

Reply:
linking and everything is not necessary because we are published right ? However what if we are making local changes to the client sdk or backend sdk, I think it will be a little difficult to set that up if our current dependencies point at the published ones rather than local ones

Reply:
Danielle and I just talked about it and I think we're going to stick with running npm link manually again for changes to the client sdk or backend sdk, since publishing to npm registry every time there's a small change would be a lot. I'll re-add the scripts in the package json